en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown

```diff
@@ -80,7 +79,7 @@ To blacklist Elasticsearch 6,
         "com.liferay.portal.search.elasticsearch6.impl", \
         "com.liferay.portal.search.elasticsearch6.spi", \
         "com.liferay.portal.search.elasticsearch6.xpack.security.impl", \
-        "Liferay Connector to X-Pack Security [Elastic Stack 6.x] - Impl" \
+        "Liferay Enterprise Search Security  - Impl" \
```

Comment:
We should keep the old name too because customers may deploy an old LPKG they downloaded before the app was renamed. @rbohl  @sez11a

Reply:
```properties
blacklistBundleSymbolicNames=[ \
	"com.liferay.portal.search.elasticsearch6.api", \
	"com.liferay.portal.search.elasticsearch6.impl", \
	"com.liferay.portal.search.elasticsearch6.spi", \
	"com.liferay.portal.search.elasticsearch6.xpack.security.impl", \
	"Liferay Connector to X-Pack Security [Elastic Stack 6.x] - Impl", \
	"Liferay Enterprise Search Security - Impl.lpkg" \
]
```

Reply:
Hey @sez11a or @rbohl

Did you make this update above ☝️ when merging?

Reply:
I merged whatever was in the branch. If a further update is needed, please send me a pull.

Reply:
https://github.com/sez11a/liferay-docs/pull/4892